Dockerfile

```diff
@@ -128,6 +139,30 @@ COPY ./docs /usr/src/code/docs
 COPY ./public /usr/src/code/public
 COPY ./src /usr/src/code/src
 
+RUN \
```

Comment:
We don't need to re-build the extension again on this image stage. Please check my previous comment about the phpredis extension and how we copy .so file between stages to reduce dependencies and final image size.

Reply:
yes, i am still working on Dockerfile